matchmake-extension/utility.go

```diff
@@ -0,0 +1,32 @@
+package matchmake_extension
+
+import (
+	//nex "github.com/PretendoNetwork/nex-go"
+	match_making "github.com/PretendoNetwork/nex-protocols-go/match-making"
+	//"github.com/PretendoNetwork/plogger-go"
+    "reflect"
```

Comment:
Maybe this is GitHubs code viewer playing tricks on me but is this line indented differently than the other stuff?